src/CatalogService.Core/Models/ItemModel.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace CatalogService.Core.Models;
+
+public record ItemModel(string Name, string? Description, string? Image, string Category, double Price, long Amount);

```

Comment:
Prefer `decimal`  for money over `double Price`, in general you might want to introduce some additional attributes like currency type.

Reply:
Regarding the Name as ID the benefit I considered here was the simplicity for the database, but you are right, if there was a typo in the Name field the only way to change it would be deleting and inserting again. I'll introduce an Id field and a uniquiness constraint for the Name field.

Reply:
For the decimal vs double field, SQLite doesn't support that by default.